js/views/modals/orderDetail/summaryTab/DisputeStarted.js

```diff
@@ -0,0 +1,72 @@
+import _ from 'underscore';
+import moment from 'moment';
+import {
+  events as orderEvents,
+} from '../../../../utils/order';
+import loadTemplate from '../../../../utils/loadTemplate';
+import BaseVw from '../../../baseVw';
+
+export default class extends BaseVw {
+  constructor(options = {}) {
+    super(options);
+
+    this._state = {
+      disputerName: '',
+      claim: '',
+      showResolveButton: true,
+      ...options.initialState || {},
+    };
+
+    this.listenTo(orderEvents, 'resolveDisputeComplete', () => {
+      this.setState({
+        showResolveButton: false,
+      });
+    });
+  }
+
+  className() {
+    return 'disputeStartedEvent rowLg';
+  }
+
+  events() {
+    return {
+      'click .js-resolveDispute': 'onClickResolveDispute',
+    };
+  }
+
+  onClickResolveDispute() {
+    this.trigger('clickResolveDispute');
+  }
+
+  getState() {
+    return this._state;
+  }
+
+  setState(state, replace = false, renderOnChange = true) {
```

Comment:
Yes, it probably should. I didn't initially because this is a very specific way of working that only I was using and it was evolving, but even if only I use it, it won't hurt for it to be in the baseVw and devs who aren't interested in it, could leave it be.

If you don't mind though, I think I'll leave this as a separate task to circle back to, since there's now 35 occurrences of it in the app and it's probably not super hight priority to work through this now. Going forward, I'll put setState() and getState() in the baseVw so we don't accumulate more technical debt and at some point I'll re-factor the other views.